gloo/docs/gloo_routing/virtual_services/delegation/_index.md

```diff
@@ -0,0 +1,312 @@
+---
+menuTitle: Delegation
+title: Delegating with Route Tables
+weight: 10
+description: >
+ Virtual Services can delegate ownership of configuration to Route Tables, top-level API objects which specify routes for a given domain and path prefix.
+
+---
+
+
+The Gloo Virtual Service makes it possible to define all routes for a domain on a single configuration resource.
+
+However, condensing all routing config onto a single object can be cumbersome when dealing with a large number of routes.
+
+For this reason, in v0.20.0, Gloo a feature referred to as *delegation*. Delegation allows routing to be configured by from a tree of config objects. The tree always has a *Virtual Service* as its root, which delgates to any number of *Route Tables*. Route Tables  can further delegate to other Route Tables.
```

Comment:
maybe "Gloo introduces a feature referred to as delegation"? Also the next sentence "configured by from a tree" reads weird.